<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriletM</class>
 <widget class="QWidget" name="FriletM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>500</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>文件传输自由「教师端」</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pic.qrc">
    <normaloff>:/Frilet.ico</normaloff>:/Frilet.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="FileSendList_tw"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>班级：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="grade_cmb">
       <item>
        <property name="text">
         <string>高一</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高二</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高三</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="class_spb"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>班</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>科目：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="subject_cmb">
       <item>
        <property name="text">
         <string>公共</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>语文</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>数学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>英语</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>物理</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>化学</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>生物</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>地理</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>政治</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>历史</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="FileSend_Btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BatchCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="select_Btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="paste_Btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>粘贴</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="AutoStart_chk">
       <property name="text">
        <string>自启动</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../pic.qrc"/>
 </resources>
 <connections/>
</ui>
